packages/services/f-feature-management/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.11.0
+------------------------------
+*February 21, 2023*
+
+### Removed
+- feature-management as a dependency and moved to monorepo root `package.json` to solve build errors
```

Comment:
What was the build issue you were seeing that this solved? (would be good to give a bit more context in the `changelog`)